### ISSUE TITLE
Claude permission_error

### ISSUE THREAD

**Issue by Rapahannock** [chathub-dev/chathub#975]
**Describe the bug**

Claude produces the following when searched:

{"error":{"type":"permission_error","message":"Invalid model","code":"model_not_allowed"}}

**Desktop (please complete the following information):**
 - OS: Windows 11, Android
 - Browser: Chrome, Lemur
 - ChatHub Version: 1.45.4
![Screenshot 2023-12-22 162550](https://github.com/chathub-dev/chathub/assets/45186344/1a6da075-3d0e-49a8-899c-c82c84d0a551)

**Comment by nolecruiser** [comment]
{"error":{"type":"permission_error","message":"Invalid model","code":"model_not_allowed"}}

I can not use Claude. ( I purchased ChatHub program)

**Comment by wong2** [comment]
Are you using API mode for Claude?

**Comment by Rapahannock** [comment]
> Are you using API mode for Claude?

I'm using Webapp mode. I don't have an API key nor access to Claude Console. POE mode works however. [Similar issue](https://github.com/chathub-dev/chathub/issues/915) happened last month. Thanks

**Comment by nolecruiser** [comment]
I am using webmode.

2023년 12월 23일 (토) 오후 11:38, wong2 ***@***.***>님이 작성:

**Comment by wong2** [comment]
@Rapahannock @nolecruiser Can you chat normally at claude.ai ?

**Comment by Rapahannock** [comment]
> @Rapahannock @nolecruiser Can you chat normally at https://claude.ai ?

Yes its fine when chatting directly from the claude.ai site. Also can, using OpenRouter and POE mode, but with Webapp mode I get the error.

**Comment by wong2** [comment]
@Rapahannock Unfortunately I can't reproduce this myself. Do you know how to debug HTTP request with Chrome devtools?

**Comment by Rapahannock** [comment]
**Get the following API Request-**

**Headers**:

Request URL: https://claude.ai/api/append_message
Request Method: POST
Status Code: 403 Forbidden
Remote Address: 160.79.104.10:443
Referrer Policy: strict-origin-when-cross-origin

**Payload**:

{organization_uuid: "1ad7b662-d56d-444d-8d2d-14ba4436bb56",…}
attachments: []
completion
: {prompt: "test", model: "claude-2.1"}
conversation_uuid: "f2bbe4b2-ab96-4d6a-9563-6453741fab24"
organization_uuid: "1ad7b662-d56d-444d-8d2d-14ba4436bb56"
text: "test"

**Preview**:

{error: {type: "permission_error", message: "Invalid model", code: "model_not_allowed"}}
error: {type: "permission_error", message: "Invalid model", code: "model_not_allowed"}

**Response**:

{
    "error": {
        "type": "permission_error",
        "message": "Invalid model",
        "code": "model_not_allowed"
    }
}

**Comment by wong2** [comment]
@Rapahannock Great! Can you do the same on https://claude.ai and debug the request, so we can compare what's the difference.

**Comment by Rapahannock** [comment]
Got the following from https://claude.ai/. Thanks

**Headers**:

Request URL: https://claude.ai/api/append_message
Request Method: POST
Status Code: 200 OK
Remote Address: 160.79.104.10:443
Referrer Policy: strict-origin-when-cross-origin

**Payload**:

{completion: {prompt: "test", timezone: "America/New_York", model: "claude-2.0"},…}
attachments: []
completion: {prompt: "test", timezone: "America/New_York", model: "claude-2.0"}
conversation_uuid: "24d949b4-0a45-4846-91da-f0f68b699720"
organization_uuid: "1ad7b662-d56d-444d-8d2d-14ba4436bb56"
text: "test"

**Preview**:

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" Hello","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":"!","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" What","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" can","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" I","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" help","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" with","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":" today","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":"?","stop_reason":null,"model":"claude-2.0","stop":null,"log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

data: {"type":"completion","id":"compl_01WTSgfA8waY7nkwKz8NyD2B","completion":"","stop_reason":"stop_sequence","model":"claude-2.0","stop":"\n\nHuman:","log_id":"compl_01WTSgfA8waY7nkwKz8NyD2B","messageLimit":{"type":"within_limit"}}

**Comment by wong2** [comment]
@Rapahannock Thanks, so the difference is the model name (claude-2.1 vs claude-2.0), I will figure this out.
